.github/workflows/release.yaml

```diff
@@ -0,0 +1,44 @@
+name: NATS Server Releases
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  run:
+    name: GitHub Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "stable"
```

Comment:
Will there be a need to pin the version? Meaning, suppose that we are at Go 1.23.x at the moment when releasing NATS v2.10.26 but then the Go stable release becomes Go 1.24, and we need to release v2.10.27, I think we may want to stay in the same Go release version to minimize the changes, no?

Reply:
If we backported this to the 2.10.x branch then I'd likely pin Go 1.23 at the same time, but not sure it's really necessary for `main`.